packages/web-forms/src/components/controls/RankControl.vue

```diff
@@ -0,0 +1,246 @@
+<script setup lang="ts">
+import ControlText from '@/components/ControlText.vue';
+import ValidationMessage from '@/components/ValidationMessage.vue';
+import type { RankNode } from '@getodk/xforms-engine';
+import type { Ref } from 'vue';
+import { computed, inject, ref } from 'vue';
+import { VueDraggable } from 'vue-draggable-plus';
+
+interface RankControlProps {
+	readonly question: RankNode;
+}
+
+interface HighlightOption {
+	index: Ref<number | null>;
+	timeoutID: NodeJS.Timeout | null;
+}
+
+const props = defineProps<RankControlProps>();
+const HOLD_DELAY = 200; // Delay in ms to hold an item before dragging, avoids accidental reordering on swipe.
```

Comment:
No need for change now unless you feel like it, but I'd usually make this a JSDoc comment so it's also available for references inline.

Reply:
I've converted it to JS Doc, and it shows in the IDE as info for that constant 
:) thanks for that!
<img width="300" alt="Screenshot 2025-01-31 at 2 35 05 PM" src="https://github.com/user-attachments/assets/c9c373dc-3ee6-4fde-ba9c-08e5c635b582" />